src/js/tracks/text-track-settings.js

```diff
@@ -58,7 +158,7 @@ function captionOptionsMenuTemplate(uniqueId, dialogLabelId, dialogDescriptionId
               </select>
             </span>
           </fieldset>
-          <fieldset class="window-color vjs-tracksetting">
+          <fieldset class="vjs-window-color vjs-tracksetting">
```

Comment:
No, but we weren't using that selector and it could pretty easily be considered a bug because all our classes should be prefixed with `vjs-`.